docker-compose-postgres.yml

```diff
@@ -0,0 +1,66 @@
+services:
```

Comment:
You can actually just overwrite the existing docker-compose.yml with these changes rather than create a separate file. Our intention is that running with postgres is the default, we just hadnt got around to making these changes yet as our underlying dockerfiles are going through a period of change and it wasnt a priority for us. But there's no value in the old one so feel free to make this the primary! :)

Reply:
Done.